src/theme/ItaliaTheme/_common.scss

```diff
@@ -43,4 +43,10 @@
       }
     }
   }
+
+  @media print {
+    .gdpr-privacy-show-banner {
+      visibility: hidden;
+    }
+  }
```

Comment:
C'è un file dedicato per gli stili di stampa, sposta in quello. Si trova in `theme/ItaliaTheme/Print/_all_pages.scss`

Reply:
Fatto!